Formula/iblinter.rb

```diff
@@ -0,0 +1,18 @@
+class Iblinter < Formula
+  desc "Linter tool for Interface Builder"
+  homepage "https://github.com/IBDecodable/IBLinter"
+  url "https://github.com/IBDecodable/IBLinter.git",
+      :tag      => "0.4.20",
+      :revision => "4b518dafbd64a329374f285c5f27bc6c08be7a38"
+  head "https://github.com/IBDecodable/IBLinter.git"
+  depends_on :xcode => ["10.2", :build]
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    # Test by showing the help scree
+    system "#{bin}/iblinter", "help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.